### ISSUE TITLE
unused variable, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1197]
```abap
DATA lv_name TYPE string VALUE 'TADIR-DEVCLASS'.
SELECT-OPTIONS s_devcl FOR (lv_name).
CLEAR s_devcl.
```
